py3status/modules/google_calendar.py

```diff
@@ -0,0 +1,521 @@
+# -*- coding: utf-8 -*-
+"""
+Display upcoming Google Calendar events.
+
+This module will display information about upcoming Google Calendar events in
+one of two formats which can be toggled with a button press. The
+event's URL may also be opened in a web browser with a button press.
+
+Configuration parameters:
+    auth_token: The path to where the access/refresh token will be saved
+        after successful credential authorization.
+        (default '~/.config/py3status/google_calendar.auth_token')
+    blacklist_events: Event names in this list will not be shown in the module
+        (case insensitive).
+        (default [])
+    button_open: Opens the URL for the clicked on event in the default
+        web browser.
+        (default 3)
+    button_refresh: Refreshes the module and updates the list of events.
+        (default 2)
+    button_toggle: The button used to toggle between output formats
+        format_event and format_event_full.
+        (default 1)
+    cache_timeout: How often the module is refreshed in seconds
+        (default 60)
+    client_secret: the path to your client_secret file which
+        contains your OAuth 2.0 credentials.
+        (default '~/.config/py3status/google_calendar.client_secret')
+    force_lowercase: Sets whether to force all event output to lower case.
+        (default False)
+    format: The format for module output.
+        (default '{events}| \?color=event \u2687')
+    format_date: The format for date related format placeholders. May be any Python
+        strftime directives for dates.
+        (default '%a %d-%m')
+    format_event: The format for the first toggleable event output.
+        (default '[\?color=event {summary}][\?if=location  ({location})]
+        [\?if=time_to  {time_to}]')
```

Comment:
Above...  No space. ----> `'{events}|\?color=event \u2687'`
This line... Try `[]`. ----> `'[\?color=event {summary}][ ({location})][ {time_to}]'`